### ISSUE TITLE
Unable to install aws-amplify: @aws-amplify/analytics@5.2.9 is not on npm

### ISSUE THREAD

**Issue by manics** [aws-amplify/amplify-js#9934]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-js/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-js/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### JavaScript Framework

Not applicable

### Amplify APIs

Not applicable

### Amplify Categories

Not applicable

### Environment information

<details>

```
# Put output below this line
npx: installed 1 in 1.024s

  System:
    OS: Linux 5.10 Amazon Linux 2
    CPU: (2) x64 Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
    Memory: 2.64 GB / 3.83 GB
    Container: Yes
    Shell: 4.2.46 - /bin/bash
  Binaries:
    Node: 14.17.0 - ~/.nvm/versions/node/v14.17.0/bin/node    npm: 6.14.13 - ~/.nvm/versions/node/v14.17.0/bin/npm
  npmGlobalPackages:    npm: 6.14.13
    pnpm: 5.18.9
    serverless: 3.8.0
```

</details>

### Describe the bug

Installing the current version of `aws-amplify` fails due to a dependency on `@aws-amplify/analytics@5.2.9` which is not published to NPM.

### Expected behavior

`npm install aws-amplify` should install without errors.

### Reproduction steps

```
[ec2-user@ip-10-0-0-8 ~]$ mkdir amplify-test

[ec2-user@ip-10-0-0-8 ~]$ cd amplify-test/

[ec2-user@ip-10-0-0-8 amplify-test]$ npm install aws-amplify
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @aws-amplify/analytics@5.2.9.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'aws-amplify'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ec2-user/.npm/_logs/2022-05-25T09_47_51_429Z-debug.log

[ec2-user@ip-10-0-0-8 amplify-test]$ npm install aws-amplify@4.3.24
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @aws-amplify/analytics@5.2.9.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'aws-amplify'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ec2-user/.npm/_logs/2022-05-25T09_48_09_870Z-debug.log

[ec2-user@ip-10-0-0-8 amplify-test]$
```

### Code Snippet

```javascript
// Put your code below this line.

```

### Log output

<details>

```
// Put your logs below this line


```

</details>

### Additional information and screenshots

`aws-amplify@4.3.24` requires `@aws-amplify/analytics@5.2.9`
https://github.com/aws-amplify/amplify-js/blob/aws-amplify%404.3.24/packages/aws-amplify/package.json#L37

But this is not currently (2022-05-25T09:53:30+00:00) available on NPM, the published version is `5.2.8`:
https://www.npmjs.com/package/@aws-amplify/analytics

Note there was another issue yesterday regarding a _previous_ version of aws-amplify: https://github.com/aws-amplify/amplify-js/issues/9930

**Comment by rohitmpatil-iprogrammer** [comment]
Is anyone looking into this?

**Comment by rohitmpatil-iprogrammer** [comment]
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @aws-amplify/analytics@5.2.9.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'aws-amplify'
npm ERR! notarget

**Comment by Rafat97** [comment]
I am trying to build a docker image. But I can't. 

When, I was trying to build image. always gives me this error.  error Couldn't find any versions for "@aws-amplify/analytics" that matches "5.2.9" 

![unknown](https://user-images.githubusercontent.com/21246862/170258240-b551a7cc-7cf0-44ea-b777-adc3600e811f.png)

Error came Because for- 
https://github.com/aws-amplify/amplify-js/blob/9e3792f60a6bcc45588d17a74713832313b2ecdc/packages/aws-amplify/package.json#L37

But, There is no version `5.2.9` -

![image](https://user-images.githubusercontent.com/21246862/170258490-c17edea5-d239-4b74-abea-cbecea2a5080.png)

**Comment by kingferiol** [comment]
It is the same here

**Comment by lucas-adamswealth** [comment]
Same issue here, even with older versions of @aws-amplify/ui-react

**Comment by ais047** [comment]
Quick fix that's working for me is to lock your package.json to 4.3.23
`"aws-amplify":` "^4.3.0" =>  "aws-amplify":` "4.3.23"`

**Comment by axelra82** [comment]
Can confirm same issue with `"aws-amplify": "^4.3.3"`.
@ais047's answer solved it for me as a temporary fix.
> Quick fix that's working for me is to lock your package.json to 4.3.23
"aws-amplify": "^4.3.0" => "aws-amplify": "4.3.23"

**Comment by jimenaCabrejas** [comment]
I had the same problem. Solved adding to package.json

`  "overrides": {
    "aws-amplify": {
      "@aws-amplify/analytics": "5.2.8"
    }
  },`

**Comment by Ryuuji159** [comment]
> I had the same problem. Solved adding to package.json
> 
> `"overrides": { "aws-amplify": { "@aws-amplify/analytics": "5.2.8" } },`

That only works with npm ^8.3.0

**Comment by albusphan** [comment]
Think the amplify remove the 5.2.9 from npm.
All my amplify crashed, please looking to this please

**Comment by iartemiev** [comment]
The Amplify team published `@aws-amplify/analytics@5.2.9` yesterday. We did not remove it. We're working with NPM to understand how it could have disappeared from the registry.

We've updated the `latest` tag in NPM to point to the previous version of aws-amplify - 4.3.23 and have confirmed that `npm i aws-amplify` is now working as expected. Please try re-installing. 

If you have version `aws-amplify@4.3.24` explicitly listed in your package.json, please replace it with 4.3.23 and run `npm install` again.
If you're still experiencing issues after that, run `rm -rf node_modules && npm install`

**Comment by thomasturrell** [comment]
> In the meantime, we've updated the `latest` tag in NPM to point to the previous version of aws-amplify - 4.3.23 

@iartemiev should this change be reverted now? i.e. latest should point to 4.3.24
